body/test/test_dynamixel_XL430.py

```diff
@@ -11,6 +11,106 @@
 
 class TestDynamixelXL430(unittest.TestCase):
 
+    def test_set_get_float(self):
+        """
+        Verify that passing floats to API doesn't crash and that can set/get values consistently
+        """
+        print('Testing Set Get Float')
+        usb = "/dev/hello-dynamixel-head"
+        dxl_id = 12
+        baud = stretch_body.dynamixel_XL430.DynamixelXL430.identify_baud_rate(dxl_id, usb)
+        m = stretch_body.dynamixel_XL430.DynamixelXL430(dxl_id,usb, baud=baud)
+        if m.startup():
```

Comment:
This test could succeed without testing anything if the Dxl fails to startup. Use self.assertTrue(m.startup()) at the beginning to fail early.